WebRoot/js/zimbraMail/share/view/ZmImportView.js

```diff
@@ -154,4 +156,11 @@ ZmImportView.prototype._handleFileChange = function(file) {
 	if (type) {
 		this.set("TYPE", type);
 	}
+
+	//show uploaded file-name in custom file-upload widget
+	var fileNameWrapper = Dwt.byId(ZmImportView.ID + "_FILE_NAME", this.getControl("FORM"));
+	if(fileNameWrapper) {
```

Comment:
You mean space before `{`. 

If so, it's consistent with others. If you look around: https://github.com/Zimbra/zm-web-client/blob/9be691c62c51a56f0b61ffb9a28e1b706cd47ce7/WebRoot/js/zimbraMail/share/view/ZmImportView.js#L152